quirks/websites-with-shared-credential-backends.json

```diff
@@ -250,6 +313,10 @@
         "go.com",
         "disneyplus.com"
     ],
+    [
+        "sodexo.com",
```

Comment:
Where can I log in on www.sodexo.com?

Reply:
Couldn't find a login form either... removed it then. Sorry about that.